src/helpers/adapters/mysql.ts

```diff
@@ -155,7 +155,20 @@ export async function storeVote(space, body, ipfs, receipt, id) {
     vp_state: ''
   };
 
-  await db.queryAsync('INSERT IGNORE INTO votes SET ?', params);
+  const connection = await db.getConnectionAsync();
+
+  await connection.beginTransactionAsync();
+
+  const deleteVotesQuery =
+    'DELETE FROM votes WHERE voter = ? AND proposal = ?;';
```

Comment:
This wont work if someone broadcast a vote with lower timestamp. We will accept lower timestamp as final vote while it should be the one with highest timestamp